fury-on-ovhcloud/README.md

```diff
@@ -0,0 +1,539 @@
+
+- `kube`: The Managed Kubernetes Cluster, such as its name and essentially network informations.
+
+- `pool`: The Kubernetes node pool parameters.
+
+Deploy the infrastructure with:
+
+```bash
+cd infrastructure/terraform
+
+terraform init
+terraform plan -var-file=variables.tfvars
+terraform apply -var-file=variables.tfvars
+```
+
+Wait a few minutes until the end of the deployement.
+
+### Configure your kubectl environment
+
+Once the Managed Kubernetes Cluster created, in the `terraform` folder, you get the associated `kubeconfig` file. Set the KUBECONFIG variable value like this:
+
+```bash
+export KUBECONFIG="$(pwd)/kubeconfig"
+```
+
+Your `kubectl` cli is ready to use.
+
+### Kustomize project
+
+Kustomize allows to group together related Kubernetes resources and combines them to create more complex deployments. 
+Moreover, it is flexible, and it enables a simple patching mechanism for additional customization.
+
+To deploy the Fury distribution, use the following root `kustomization.yaml` located `manifests/kustomization.yaml`:
+
+```yaml
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ingress
+  - logging
+  - monitoring
+```
+
+This `kustomization.yaml` wraps the other `kustomization.yaml`s in subfolders. For example in `manifests/logging/kustomization.yaml`
+
+```yaml
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../vendor/katalog/logging/cerebro
+  - ../../vendor/katalog/logging/logging-operator
+  - ../../vendor/katalog/logging/logging-operated
+  - ../../vendor/katalog/logging/configs
+  - ../../vendor/katalog/logging/opensearch-single
+  - ../../vendor/katalog/logging/opensearch-dashboards
+
+  - resources/ingress.yml
+
+patchesStrategicMerge:
+  - patches/opensearch-resources.yml
+  - patches/cerebro-resources.yml
+```
+
+Each `kustomization.yaml`:
+
+- references the modules downloaded in the previous section
+- patches the upstream modules (e.g. `patches/opensearch-resources.yml` limits the resources requested by OpenSearch)
+- deploys some additional custom resources (e.g. `resources/ingress.yml`)
+
+Install the modules:
+
+```bash
+cd manifests/
+
+make apply
+# Due to some chicken-egg 🐓🥚 problem with custom resources you have to apply multiple times
+make apply
+```
+
+## Step 4 - Explore the distribution
+
+🚀 The distribution is finally deployed! In this section, you explore some of its features.
+
+### Setup local DNS
+
+In Step 3, alongside the distribution, you have deployed Kubernetes ingresses to expose underlying services at the following HTTP routes:
+
+- `forecastle.fury.info`
+- `grafana.fury.info`
+- `opensearch-dashboards.fury.info`
+
+To access the ingresses more easily via the browser, configure your local DNS to resolve the ingresses to the external load balancer IP:
+
+1. Get the address of the external load balancer:
+
+```bash
+kubectl get svc -n ingress-nginx ingress-nginx --no-headers | awk '{print $4}'
```

Comment:
I NEED TO TEST THIS, but I'm 99% sure it works

No need to pipe to `awk`, you can tell kubectl what to print out using the `-o jsonpath=<template>` flag:
```suggestion
kubectl get svc -n ingress-nginx ingress-nginx -ojsonpath='{.spec.externalIPs[*]}'
```